pkg/resource/event_bus/custom_update_api.go

```diff
@@ -0,0 +1,146 @@
+package event_bus
```

Comment:
I actually think it's best to just put all the custom code into `hooks.go`...

Reply:
Done